deploy/kubernetes/webhook-example/webhook.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: snapshot-validation-deployment
+  namespace: default # NOTE: change the namespace
```

Comment:
Should this namespace be the same as the namespace of the snapshot-controller?

Reply:
It doesn't need to be, but it may be easier to organize that way.